### PR TITLE
Update yarn lock and cleaning

### PR DESCRIPTION
**Pull request by bidoubiwa**
## In this PR 

- add build in eslintignore to avoid linting the react build folder
- Add `playground build` job workflow that checks if the react and vue playground build correctly
- Add the job to the bors toml 
- Add vue file in the lining command to avoid ignoring them on lint
- Add two scripts in package json to launch the playground tests
- Remove eslint from `vue` playground because it inherits them from root directory
- Update yarn locks

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,9 +9,11 @@
     "test:demo:browser": "yarn --cwd examples/express && yarn --cwd examples/express test",
     "test:demo:nodejs": "node examples/node/index.js",
     "test:demo:esm": "yarn --cwd examples/esm && yarn --cwd examples/esm start",
+    "test:playground:vue": "yarn --cwd ./playgrounds/vue && yarn --cwd ./playgrounds/vue build",
```
Comment (curquiza) [inline package.json:12]: It'snot really a test, but a build:

```suggestion
    "build:playground:vue": "yarn --cwd ./playgrounds/vue && yarn --cwd ./playgrounds/vue build",
```

File: package.json [hunk 2]
```diff
@@ -9,9 +9,11 @@
...
+    "test:playground:react": "yarn --cwd ./playgrounds/react && yarn --cwd ./playgrounds/react build",
```
Comment (curquiza) [inline package.json:13]:
```suggestion
    "build:playground:react": "yarn --cwd ./playgrounds/react && yarn --cwd ./playgrounds/react build",
```

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -41,3 +41,16 @@ jobs:
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install dependencies
+        run: yarn install
+      - name: Run tests Vue Build
+        run: yarn test:playground:vue
+      - name: Run tests React Build
+        run: yarn test:playground:react
```
Comment (curquiza) [inline .github/workflows/test.yml:56]:
```suggestion
      - name: Run Vue build
        run: yarn build:playground:vue
      - name: Run React build
        run: yarn build:playground:react
```

File: bors.toml [hunk 4]
```diff
@@ -1,3 +1,3 @@
-status = ["instant-meilisearch-tests", "instant-meilisearch-style-tests"]
+status = ["instant-meilisearch-tests", "instant-meilisearch-style-tests", "instant-meilisearch-playground-builds"]
```
Comment (curquiza) [inline bors.toml:1]:
```suggestion
status = ['integration-tests', 'style-check', 'playground-builds']
```

Comment (curquiza) [reply bors.toml:1]: Can you rename the job like this please? It's more readable, too long name are not really convenient to read on the PR 🙂
Sorry for this commit suggestion, it's not really related to your PR, I can do it myself if you want. It also makes you change the name in the CI file.

**Review by curquiza — CHANGES_REQUESTED** [review]
Thanks for this PR.

File: bors.toml (hunk 4 above)
Comment (bidoubiwa) [reply bors.toml:1]: I can do it no problem

File: bors.toml [hunk 5]
```diff
@@ -1,3 +1,3 @@
-status = ["instant-meilisearch-tests", "instant-meilisearch-style-tests"]
+status = ["integration-tests", "lint-check", "playground-builds"]
```
Comment (curquiza) [inline bors.toml]:
```suggestion
status = ['integration-tests', 'lint-check', 'playground-builds']
```

**Review by curquiza — APPROVED** [review]
Just a small change, then I approve!
merge when you want

**Comment by curquiza** [comment]
bors try

File: bors.toml [hunk 6]
```diff
@@ -1,3 +1,3 @@
-status = ["instant-meilisearch-tests", "instant-meilisearch-style-tests"]
+status = ["integration-tests", "style-check", "playground-builds"]
```
Comment (curquiza) [inline bors.toml:1]:
```suggestion
status = ['integration-tests', 'style-check', 'playground-builds']
```

**Review by curquiza — APPROVED** [review]

**Review by curquiza — APPROVED** [review]

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**